src/plugins/vis_type_timeseries/public/application/components/annotations_editor.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useCallback } from 'react';
+import uuid from 'uuid';
+import { EuiSpacer, EuiTitle, EuiButton, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { AnnotationRow } from './annotation_row';
+import { collectionActions, CollectionActionsProps } from './lib/collection_actions';
+
+import type { Panel, Annotation } from '../../../common/types';
+import type { VisFields } from '../lib/fetch_fields';
+
+interface AnnotationsEditorProps {
+  fields: VisFields;
+  model: Panel;
+  onChange: (partialModel: Partial<Panel>) => void;
+}
+
+export const newAnnotation = () => ({
+  id: uuid.v1(),
+  color: '#F00',
+  index_pattern: '',
+  time_field: '',
+  icon: 'fa-tag',
+  ignore_global_filters: 1,
+  ignore_panel_filters: 1,
+});
+
+const NoContent = ({ handleAdd }: { handleAdd: () => void }) => (
+  <EuiText textAlign="center">
+    <p>
```

Comment:
Nit, do we need the extra <p> ?

Reply:
Yes... don't see any difference with and without `p`

Reply:
I was wrong it affect UI, let's keep it as before

![image](https://user-images.githubusercontent.com/20072247/122559862-8c008880-d048-11eb-9f27-a0cd48a06745.png)